account_banking_pain_base/models/account_payment_order.py

```diff
@@ -230,7 +194,7 @@ def generate_start_payment_info_block(
             instruction_priority = etree.SubElement(
                 payment_type_info, 'InstrPrty')
             instruction_priority.text = priority
-        if self.sepa:
+        if eval_ctx.get('sepa'):
```

Comment:
Is it necessary to pass this flag via eval_ctx? Is it not better to pass it as a function argument?

Reply:
I did this to avoid changing the signature of the function, which could be a problem for modules using bank-payment like l10n modules...

But If you think I can do it, it'll be probably better to do it with an argument, yes.

Reply:
Well, the breaking change is there anyway (one needs to pass sepa in the context for it to work), so adding an additional keyword argument is going to be more readable.